OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -828,6 +844,22 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
     thread_name.set_timestamp_ns(event->GetTimestamp());
     listener_->OnThreadName(std::move(thread_name));
 
+  } else if (is_tracepoint) {
+    absl::flat_hash_map<uint64_t, orbit_grpc_protos::TracepointInfo>::const_iterator it =
+        stream_id_to_tracepoint_.find(stream_id);
```

Comment:
I'd suggest
`auto it = stream_id_to_tracepoint_.find(stream_id);`

Reply:
Thank you!